CHANGELOG.md

```diff
@@ -82,6 +82,8 @@
   [#9903](https://github.com/Kong/kong/pull/9903)
 
 ### Fixes
+- **gRPC gateway**: Fix an issue where handling `null` in JSON payload
+  [#10687](https://github.com/Kong/kong/pull/10687)
```

Comment:
Please be specific about what has been fixed.  "Fix an issue" is not helpful.

Reply:
updated, PTAL